darts/explainability/tft_explainer.py

```diff
@@ -0,0 +1,249 @@
+from typing import Dict, List, Literal, Optional
```

Comment:
Could you add a docstring header in the file, so that it gets rendered along with the rest of the doc?
You can also put some high level explanations about what it does here, and point to the key methods. Even better: provide code snippets showing how to use it.
You can have a look at `shap_explainer.py` for an example of such a header.

Reply:
On another front, `Literal` is not available in Python 3.7, you can use this instead:
```python
try:
    from typing import Literal
except ImportError:
    from typing_extensions import Literal
```

Reply:
I changed the import as you suggested but I am now receiving a linting error.

Do you know why I get the linting error at my newly added code: https://github.com/Cattes/darts/blob/feature/675_tft_explainer/darts/explainability/tft_explainer.py#L14
while the linter is not complaining about it at other pieces of code that are older like in `utils/data/tabularization.py` for example?
https://github.com/Cattes/darts/blob/feature/675_tft_explainer/darts/utils/data/tabularization.py#L6

Reply:
Yes, this is because your new import (line 14) is correct, but you need to remove `Literal` from line 1.

Reply:
Thank you. I have removed the line from the code and it's now working. 
I had the docstrings uncommited locally and overlooked the removal of the `Literal` from the `typing` module import when I pushed my code.